docs/connectors/sinks/postgresql.md

```diff
@@ -0,0 +1,256 @@
+# PostgreSQL Output Connector
+
+:::caution Experimental feature
+PostgreSQL support is an experimental feature of Feldera.
+:::
+
+Feldera allows you to output data from a SQL view to a PostgreSQL database.
+These outputs are made as a series of transactions.
+
+:::important
+Only SQL views with [Uniqueness Constraints](/connectors/unique_keys) can output
+data to a PostgreSQL table.
```

Comment:
Do we require or recommend the Postgres table to have a PK? It might be important for performance.

Reply:
Well, yes, it is required that the Postgres table have a PK, the same column that is the index in the output view specially for deletion.